css/css-anchor-position/anchor-position-dynamic-004.html

```diff
@@ -38,8 +38,8 @@
          data-offset-x=50 data-offset-y=10
          data-expected-width=30 data-expected-height=20></div>
   </div>
-<script>
+<script type="module">
```

Comment:
Why do a few of them need to be a module script? If it's necessary only for the polyfill, it's likely a good idea to add a comment to those indicating why it's a module. Otherwise, someone will come along and remove it later.

Reply:
Modules are required for top-level [await](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/await). Should we add a comment to each of these?

Reply:
I'll note that there are other WPTs (e.g. css-grid) using `<script type="module">` with top-level `await`, and no explanatory comments that I can find. 🤷